src/main/menu_app/src/App.js

```diff
@@ -12,16 +12,16 @@ import './App.css';
 const App = () => {
   return (
     <div className="app">
-        <Router>
-          <Navbar />
-          <div data-bs-spy="scroll" data-bs-root-margin="0px 0px -40%" data-bs-smooth-scroll="true" className="scrollspy-example bg-body-tertiary p-3 rounded-2" tabIndex="0">
+      <Router>
```

Comment:
Feels like this commit should be fixed up with https://github.com/KateLocate/menu_generator/pull/6/commits/d0e076f21a11d7682cf49610852b4c84aba752fa, no? Looks like they do relatively connected things